samples/Beyer2024/globular_protein.py

```diff
@@ -87,6 +87,8 @@
 inputs={"pH": args.pH,
         "pdb": args.pdb}
 
+steps_name = f"{espressomd.version.friendly() == '4.2' and 'reaction_' or ''}steps"
```

Comment:
While this is a very compact and elegant way of solving the change in the API of espresso, it might be a bit cryptic for people missing the context. What about expanding the code a bit and adding a clarifying comment?
```python
# To keep backwards compatibility due to an API change from espresso 4.2 to espresso 4.3
if espressomd.version.friendly() == '4.2':
       keyword_mc_reaction="reaction_steps"
else:
       keyword_mc_reaction="reaction"
```
If you agree then the function should also be changed in the other sample scripts for consistency.

Reply:
I considered it, but was faced with the issue of code coverage decreasing, since we don't run CI with ESPResSo 4.3-dev yet. If you're fine with that, I will apply the changes you propose.

I was also wondering whether to implement free functions like `setup_electrostatic_interactions()` that take the system as argument and the solver or particle type, and let this function go through the if/else for us. This way we reduce code duplication and make pyMBE compatible with multiple ESPResSo releases. The downside is that we end up creating a new ESPResSo interface that pyMBE devs now have to maintain.

Reply:
I was just reflecting about the same actually.  To me it boils down to the same dilemma than with the rest of "convenience" functions  in `lib/handy_functions.py`: we reduce code repetition at the cost of creating an extra layer on top of espresso that is arguably not needed and sometimes more limited that the native functions. I guess that If we give the added value of helping our users with backwards compatibility with earlier versions of ESPResSo, it could make some sense to keep it. I have not yet came to an absolute decision on this topic, but I suggest that for consistency with the current version we go for the strategy of having those functions in `libs/handy_functions.py`. In the long run, if we decide to keep that library we then need to integrate it into our CI pipeline and maintain it properly.

Reply:
Done.